client/webpack.config.js

```diff
@@ -1,5 +1,5 @@
 /**
- * **** WARNING: No ES6 modules here. Not transpiled! ****
+ * WARNING: No ES6 modules here. Not transpiled! ****
```

Comment:
Nit of the day: if we're removing the leading `****` should we also remove the trailing one?
```suggestion
 * WARNING: No ES6 modules here. Not transpiled!
```

Reply:
Done.
For the record, this was done automatically by https://github.com/gajus/eslint-plugin-jsdoc#eslint-plugin-jsdoc-rules-no-multi-asterisks, so this may surface again in future PRs